### PR TITLE
set device to UTC

### PR DESCRIPTION
**Pull request by aerickson**
Suggested solution from Bitbar. see #19 

Also:
  - refactoring: pull out ADB_COMMAND_TIMEOUT

### CODE REVIEW

File: taskcluster/script.py [hunk 1]
```diff
@@ -166,6 +165,11 @@ def main():
         print('Android device version (ro.build.version.release):  {}'.format(android_version))
         # this can explode if an unknown device, explode now vs in an hour...
         device_type = get_device_type(device)
+        # set device to UTC
+        device.shell('setprop persist.sys.timezone "UTC"', timeout=ADB_COMMAND_TIMEOUT)
```
Comment (bclary) [inline taskcluster/script.py]: Don't use shell directly, use shell_output. This didn't work on my p2 without being root so you probably need root=True here.

**Review by bclary — CHANGES_REQUESTED** [review]
r- do the root dance plz.

File: taskcluster/script.py (hunk 1 above)
Comment (aerickson) [reply taskcluster/script.py]: Thanks. Fixed.

**Review by bclary — APPROVED** [review]
r+

**Pull request merged**